newapi/ooniapi/measurements.py

```diff
@@ -1361,15 +1341,19 @@ def add_axis(axis, cols, colnames, group_by):
         group_by.append(column(axis))
 
     group_by: List = []
-    if axis_x == "measurement_start_day":
-        group_by_date(since, until, cols, colnames, group_by)
-    elif axis_x:
-        add_axis(axis_x, cols, colnames, group_by)
+    try:
+        if axis_x == "measurement_start_day":
```

Comment:
Should we perhaps return an error if the user has chosen a `group_by` value, but is not aggregating on the `measurement_start_day` axis?

Also, we probably ought to change the name of `measurement_start_day` to `measurement_start_time`, since it's no longer the `day`, but rather something that changes depending on the value of the `group_by` field.

Reply:
We discussed changing the name with @majakomel but it will take a second PR in order not to break production during the transition